integration-tests/api/__tests__/admin/order/order.js

```diff
@@ -2360,4 +2365,99 @@ describe("/admin/orders", () => {
       }
     })
   })
+
+  describe("POST /orders/:id/refund", () => {
+    const orderId = idMap.getId("refund-order-1")
+
+    beforeEach(async () => {
+      await adminSeeder(dbConnection)
+      await orderSeeder(dbConnection)
+
+      await simpleOrderFactory(dbConnection, {
+        id: orderId,
+        tax_rate: null,
+        email: "test@testson.com",
+        fulfillment_status: "fulfilled",
+        payment_status: "captured",
+        line_items: [
+          {
+            id: idMap.getId("item-1"),
+            quantity: 1,
+            fulfilled_quantity: 1,
+            shipped_quantity: 1,
+            unit_price: 1000,
+          },
+        ],
+      })
+
+      await simplePaymentFactory(dbConnection, {
+        provider_id: "test-pay",
+        order: orderId,
+        amount: 300,
+        captured: true,
+      })
+
+      await simplePaymentFactory(dbConnection, {
+        provider_id: "test-pay",
+        order: orderId,
+        amount: 700,
+        captured: true,
+      })
+    })
+
+    afterEach(async () => {
+      const db = useDb()
+      await db.teardown()
+    })
+
+    it("set status on refunded order", async () => {
+      const api = useApi()
+
+      const response = await api.post(
+        `/admin/orders/${orderId}/refund`,
+        { amount: 1000, reason: "other" },
+        {
```

Comment:
suggestion(non-blocking): Could you create a const at the top like ew are doing in the other tests file. No need to refactor the other tests but just so that it can be used when someone will refactor them